change.diff

```diff
@@ -0,0 +1,431 @@
+diff --git a/src/node/internal/internal_errors.ts b/src/node/internal/internal_errors.ts
+index 00ba8dbd5..1b70b1082 100644
+--- a/src/node/internal/internal_errors.ts
```

Comment:
Oh... heh... this diff likely shouldn't be here :-)

Reply:
Removed.